pyOSPParser/system_configuration.py

```diff
@@ -124,6 +162,16 @@ class OspInitialValue(OspSystemStructureAbstract):
     _required_keys = ['variable', 'value']
 
     def __init__(self, dict_xml=None, **kwargs):
+        """
+        Constructor for OspInitialValue. One should provide 'variable (str)' and
+        'value (float, int, str, bool)'. Otherwise, one can provide dictionary that
+        contains the information.
+
+        Args:
+            variable
```

Comment:
These arguments are not in the signature, why?

Reply:
These should be recognized in **kwargs and validated in the init using _required_keys attribute.